### DISCUSSION TITLE
Messed up the TCU

### DISCUSSION THREAD

**Discussion by fortizen** [cedricp/ddt4all#1283] [Q&A]
Hey friends, i was playing with the TCU (Telematics Control Unit) on my Megane and i pressed something like reset ecu configuration and now it shows a fault on the Dashboard "Check Emergency Call System", also the TCU is showing this Error Code "B1341". 

Can anyone help me fix it or is it unfixable?

**Comment by matzel82** [comment]
Possible you need now the automatic configuration from Renault Garage.

**Reply by fortizen** [reply]
Do you think it's Expensive to fix and should i get a Clone Can Clip Adapter and Software?

**Reply by matzel82** [reply]
Not expensiv. Dont touch Clip-Clone.  Ist dangerous. And you Need Access to Renault Servers. Clip Clone doesnt have it.
If you use this software, Renault can determine your data and find out who you are.

**Reply by fortizen** [reply]
I live in North Macedonia, what do you think or know the price range is?

**Reply by matzel82** [reply]
I dont know… i came from germany. Our Customer pay 1 Hour
